tests/assert_pass/storage_not_cleared_after_failed_call.yul

```diff
@@ -0,0 +1,9 @@
+{
+    sstore(0, 7)
+    mstore(0, 9)
+    let x := call(10000, 0, 0, 0, 0, 0, 0)
+    if iszero(x) {
+        assert(eq(sload(0), 7))
```

Comment:
This assert should fail because I just encoded "storage may be different now" and not yet "storage may be different now, but is the same if the call reverted".

Reply:
basically what I said in the channel: the second assertion makes the whole formula unsat, and both assertions go into the same formula, so you get a single `unsat` in the end. If you remove the second assertion this one fails as expected